modules/thermalmanager.dot

```diff
@@ -0,0 +1,161 @@
+/* -*- mode: c -*-
+ *
+ * Graph of thermal measurement control flow
+ *
+ * To create PNG image from this, use:
+ *
+ * % dot -T png thermalmanager.dot -o thermalmanager.png
```

Comment:
The graph was created/edited manually while making the changes to the codebase. Left it in place after finishing as I thought it might be useful reference in the future too.

But since the graph is of relatively limited interest, there is no pre-existing doc package where to use it and slowdown from adding graphviz as build dep -> not build automatically.